_includes/featuredbox.html

```diff
@@ -8,9 +8,9 @@
                     <div class="wrapthumbnail">
                         <a href="{{ site.baseurl }}{{ post.url }}">
                             {% if site.lazyimages == "enabled" %}
-                            <img class="featured-box-img-cover lazyimg"  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image contains "://" %}{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image }}{% endif %}">
+                            <img class="featured-box-img-cover lazyimg"  src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAAMAAAACCAQAAAA3fa6RAAAADklEQVR42mNkAANGCAUAACMAA2w/AMgAAAAASUVORK5CYII=" data-src="{% if post.image-location contains "://" %}{{ post.image-location}}thumbnail_{{post.image }}{% else %}{{ site.baseurl }}/{{ post.image-location }}thumbnail_{{ post.image }}{% endif %}">
                             {% else %}
-                            <img class="featured-box-img-cover" src="{% if post.image contains "://" %}{{ post.image }}{% else %}{{ site.baseurl }}/{{ post.image }}{% endif %}">
+                            <img class="featured-box-img-cover" src="{% if post.image-location contains "://" %}{{ post.image-location }}thumbnail_{{ post.image }} }}{% else %}{{ site.baseurl }}/{{ post.image-location }}thumbnail_{{ post.image }}{% endif %}">
```

Comment:
Instead of using `post.image-location`, put the image-location variable in the `_config.yml` file and use `site.post-location` instead. This is similar to pulling the `site.baseurl` variable.